docs/developers/developing_iree/sanitizers.md

```diff
@@ -1,79 +1,49 @@
-# Using Address/Memory/Thread Sanitizers
+# Using Sanitizers (ASAN, TSAN, etc)
 
-[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html),
-[MemorySanitizer](https://clang.llvm.org/docs/MemorySanitizer.html) and
-[ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html) are tools
-provided by `clang` to detect certain classes of errors in C/C++ programs. They
-consist of compiler instrumentation (so your program's executable code is
-modified) and runtime libraries (so e.g. the `malloc` function may get
-replaced).
+[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) and
+[ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html), abbreviated as "ASAN" and "TSAN" respectively, are LLVM instrumentation features helping to detect certain classes of bugs. They are available both via Clang (to build C/C++ code, such as IREE's binaries) and via `iree-compile` (to build IREE bytecode modules). This page discusses both Clang and `iree-compile` usage, and the interactions between the two, since there are consistency requirements between how IREE bytecode modules are instrumented and how the IREE runtime, which loads them, is instrumented.
 
-They are abbreviated as "ASan", "MSan" and "TSan" respectively.
+## Cheat sheet
 
-They all incur large overhead, so only enable them while debugging.
+Sanitizer to use  | ASAN    | TSAN
+----------------- | ------------------ | -----------------
+Compile my own sanitizer-instrumented bytecode module | `iree-compile --iree-llvmcpu-sanitize=address --iree-llvmcpu-link-embedded=false` . No need for ASAN instrumentation in `iree-compile` itself. | `iree-compile --iree-llvmcpu-sanitize=thread --iree-llvmcpu-link-embedded=false` . No need for TSAN instrumentation in `iree-compile` itself.
```

Comment:
(I think I gave this feedback elsewhere before...) May want to qualify these sections with "running CPU code" or "using the llvm-cpu target". Sanitizers aren't really a thing for the other compiler targets (e.g. Vulkan, WebGPU, CUDA).